### ISSUE TITLE
Redirect to url after login

### ISSUE THREAD

**Issue by Wambere** [onaio/superset-patchup#31]
After #27 it would be nice to be able to fully redirect to a url after logging in to superset

**Comment by moshthepitt** [comment]
Fixed by #29

**Comment by Wambere** [comment]
Not exactly. The fix only works for when a user is logged in, we need a fix that will redirect even when the user was not logged in

**Comment by moshthepitt** [comment]
Ah, okay
